src/dev/HUDL.cpp

```diff
@@ -1,33 +1,23 @@
-/**
-* This is a basic sample of using the UART module. The program provides a
-* basic echo functionality where the uart will write back whatever the user
-* enters.
-*/
-
 // clang-format off
-#include <EVT/dev/LCD.hpp>
-#include <EVT/io/GPIO.hpp>
-#include <EVT/io/SPI.hpp>
-#include <EVT/utils/log.hpp>
 #include <dev/HUDL.hpp>
+#include <core/utils/log.hpp>
```

Comment:
Remove the clang-format disables around the includes and re-run the auto format.